### ISSUE TITLE
Choice of hyperparameters

### ISSUE THREAD

**Issue by borisdayma** [Rudrabha/Wav2Lip#18]
Hi, first of all this is a really amazing projects!

I was wondering how you decided to set:
* `sw`, sync penalty weight = 0.03
* `sg`, adversarial loss = 0.07

Also how did you choose:
* learning rate
* any weight decay (does not seem to be any)
* any other important hyper-parameter that had a large effect

GAN's are really hard to train in my experience so I'm always interested in knowing the story of successful experiments!

**Comment by prajwalkr** [comment]
Hello! 

Typically, the losses from discriminators are weighted 1/10th the reconstruction loss weight. Keeping this range in mind, we have empirically determined these weights. 

With the Learning rate, we have not experimented much. We have tried $10^(-3)$ and $10^(-4)$ and just found the latter works better and stuck with it. 

I hope this answers your questions :-)
